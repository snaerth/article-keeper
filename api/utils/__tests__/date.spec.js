import parseDateYearMonthDay from '../date';

describe('Run tests for dates', () => {
  it('Fail for parse date to YYYY-mm-dd format', async () => {
    const date = await parseDateYearMonthDay('1983-04-27');
    expect(date).toEqual(new Date('1983-04-27T00:00:00.000Z'));
  });

  it('Parse date to YYYY-mm-dd format', async () => {
    const date = await parseDateYearMonthDay('Fail-04-30');
    expect(date).toThrowErrorMatchingSnapshot();
  });
});
