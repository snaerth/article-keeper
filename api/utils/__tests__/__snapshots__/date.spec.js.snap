// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Run tests for dates Parse date to YYYY-mm-dd format 1`] = `"received is not a function"`;
